.github/workflows/Tests.yaml

```diff
@@ -66,6 +66,22 @@ jobs:
           cd $GITHUB_WORKSPACE
           ./tools/CheckFiles.sh
 
+  # Run format checker for python to make sure the code is pep8 conforming
+  # and passes yapf
+  check_python_formatting:
+    name: Check python formatting
+    runs-on: ubuntu-latest
+    container:
+      image: sxscollaboration/spectrebuildenv:latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Check python formatting
+        run: >
```

Comment:
If you make the `>` a `|` then each line below ends with a line break so you can delete the empty line 82